static/app/views/usageStats/usageStatsOrg.tsx

```diff
@@ -327,7 +327,9 @@ class UsageStatsOrganization extends AsyncComponent<Props, State> {
       return (
         <Panel>
           <PanelBody>
-            <LoadingIndicator />
+            <LoaderWrapper>
+              <LoadingIndicator />
+            </LoaderWrapper>
```

Comment:
Can we replace this with `<Placeholder>`? We should aim to eliminate spinners and instead use content placeholders as we already have too many spinners and there's a perceived perf benefit with placeholders.